src/Main.hs

```diff
@@ -180,3 +274,17 @@ main = do
             else raiseStatus status401 "Invalid Credentials"
     get "/" $ do
       json Color {peter = "peter", jonas = 3}
+    get "/recipes" $ json =<< liftIO (getAllRecipes db)
+    post "/recipes" $ do
+      recipe <- jsonData :: ActionM Recipe
+      success <- liftIO $ addRecipe db recipe
+      if success
+        then html "Recipe inserted into database!"
```

Comment:
Hier bitte status "201 created" nutzen

Reply:
9d8ac164eaaf6e8355e1254ae071ac5eb29b6aef now uses status 201 instead of html